### PR TITLE
WG EoY Document Updates

### PR DESCRIPTION
**Pull request by Cyber-JiuJiteria**
To request funding and get incrubating status, here is a pull request with the following updates:

- added a CONTRIBUTING.md file that directs folks to our slack channel and public calls
- Updated the README.md file to include links to slack and calender, YouTube playlist for DEI meetings and office hours, and directed newcomers to the CONTRIBUTING.md file to get started.

@marcelamelara @camaleon2016

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 
 <h1>Diversity, Equity and Inclusion (DEI) for Open Source Developers</h1>
```
Comment (marcelamelara) [inline README.md]: Should we remove this line since we have the new welcome below?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Contributing to the DEI Working Group
```
Comment (marcelamelara) [inline CONTRIBUTING.md]: Just so there's no ambiguity
```suggestion
# Contributing to the OpenSSF DEI Working Group
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+## How to Contribute
+
+We value all contributions, from improving documentation to participating in discussions and developing new initiatives. Here’s how you can get started:
+
+### 1. Participate in Discussions
+- Join our #wg-dei channel via the [OpenSSF Slack channel](#https://openssf.slack.com/join/shared_invite/zt-2sa4ub3pw-TKfiwHzTINSQY4ZSunhnSw#/shared-invite/email).
+- Attend working group meetings listed on the public calendar.
+
+### 2. Report Issues or Suggest Enhancements
+- Use the GitHub Issues tab to report problems or suggest ideas.
```
Comment (marcelamelara) [inline CONTRIBUTING.md]: We should probably add a link to the issues tab here.

**Review by marcelamelara — APPROVED** [review]
I just have a couple of nitpicky changes, but this looks great. Thanks @Cyber-JiuJiteria !

File: README.md (hunk 1 above)
Comment (marcelamelara) [reply README.md]: Also, I didn't realize our current README uses a mix of markdown and html. @Cyber-JiuJiteria would you mind converting the headings to markdown please?

File: README.md [hunk 4]
```diff
@@ -19,22 +21,27 @@ The scope of this Technical initiative is to focus on research, technology imple
...
+
+<h4>Communication Channels</h4>
 * Official communications occur on the DEI Working Group [mailing list](https://lists.openssf.org/g/openssf-wg-dei)
 * [Manage your subscriptions to Open SSF mailing lists](https://lists.openssf.org/g/main/subgroups)
 * Join the conversation on [Slack](https://app.slack.com/client/T019QHUBYQ3/C068TF7AH0U)
 
 <h3>Quick Start</h3>
 
-Areas that need contributions Build information if applicable Where to file issues Etc.
+Ready to contribute? Start by reviewing our [CONTRIBUTING.md](./CONTRIBUTING.md) file for guidelines on how to participate. You'll find details on connecting with us through our public calendar and Slack channel, reporting issues, suggesting improvements, and submitting pull requests. We look forward to building a more inclusive cybersecurity community together!
```
Comment (marcelamelara) [inline README.md]: Looking through this README some more, I feel like this Quick Start section is kinda buried in the middle, even though we should be highlighting it. Maybe we should move this to the top of the Get Involved section.

File: README.md [hunk 5]
```diff
@@ -1,48 +1,60 @@
...
+1. [Marcela Melara](https://github.com/marcelamelara) (Intel)
+2. [Yesenia Yser](https://github.com/Cyber-JiuJiteria) (Microsoft) 
+3. [Jay White](https://github.com/camaleon2016) (Microsoft)
 
+### Get Involved
 
-* [2023 Meeting Minutes](https://docs.google.com/document/d/1Ba_NJWWlsbTn2kVoBFpnMFMmdez9yJrHayocaxKkjaY/edit#heading=h.rmfd9b3ngn45)
+Anyone is welcome to join our open discussions related to the group's mission and charter.
+
+* [2025 Meeting Minutes]()
```
Comment (marcelamelara) [inline README.md]:
```suggestion
* [2025 Meeting Minutes](https://docs.google.com/document/d/17j8uN_radgNcY4G8u1Ua8FN__lUL4TeUN0gb-D2TrZ4/edit?tab=t.0)
```

File: README.md [hunk 6]
```diff
@@ -1,48 +1,60 @@
...
 * Official communications occur on the DEI Working Group [mailing list](https://lists.openssf.org/g/openssf-wg-dei)
-* [Manage your subscriptions to Open SSF mailing lists](https://lists.openssf.org/g/main/subgroups)
+* Manage your subscriptions to [Open SSF mailing lists](https://lists.openssf.org/g/main/subgroups)
 * Join the conversation on [Slack](https://app.slack.com/client/T019QHUBYQ3/C068TF7AH0U)
+* Explore the [GitHub Issues](https://github.com/ossf/wg-dei/issues) to collaborate 
 
-<h3>Quick Start</h3>
+### Additional Details
 
-Areas that need contributions Build information if applicable Where to file issues Etc.
```
Comment (marcelamelara) [inline README.md:32]: Do we still need this section? Looks like this content was removed.

**Comment by camaleon2016** [comment]
All looks good to me as well. Thanks for putting this together @Cyber-JiuJiteria

**Review by camaleon2016 — APPROVED** [review]

**Pull request merged**